frontend/src/FullPageRoutes/FAQ.tsx

```diff
@@ -0,0 +1,169 @@
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  makeStyles,
+  Typography,
+  Grid,
+} from "@material-ui/core"
+import * as React from "react"
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore"
+
+const useStyles = makeStyles({
+  page: {
+    margin: "1px",
+    padding: "50px",
+    backgroundColor: "rgb(238,235,228)",
```

Comment:
Just a general thought, not specific to this instance, we need to abstract out the color scheme during the CSS polish so that they're consistent across the site and give them logical names.